euth/contrib/validators.py

```diff
@@ -0,0 +1,28 @@
+from django.core.exceptions import ValidationError
+from django.utils.translation import ugettext_lazy as _
+
+image_max_mb = 5
+
+def validate_image(image, min_width, min_height):
+    errors = []
+    image_max_size = image_max_mb * 10**6
+    if image.size > image_max_size:
+        msg = _('Image should be at most {max_size} MB')
+        errors += ValidationError(msg.format(max_size=image_max_mb))
```

Comment:
You can't append to a `list` using the `+=` operator. Unfortunately this does not result in an error but in an empty list.